pkg/apis/enterprise/v2/indexercluster_types.go

```diff
@@ -106,7 +106,7 @@ type IndexerClusterStatus struct {
 // +kubebuilder:subresource:scale:specpath=.spec.replicas,statuspath=.status.replicas,selectorpath=.status.selector
 // +kubebuilder:resource:path=indexerclusters,scope=Namespaced,shortName=idc;idxc
 // +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase",description="Status of indexer cluster"
-// +kubebuilder:printcolumn:name="Master",type="string",JSONPath=".status.clusterMasterPhase",description="Status of cluster master"
+// +kubebuilder:printcolumn:name="Manager",type="string",JSONPath=".status.clusterMasterPhase",description="Status of cluster manager"
```

Comment:
This one I'm not sure about.  Whats `name` in this context? Probably ok though if all automation tests pass.

Reply:
This code is all commented so I believe it's a legacy no longer used.  I thought even better would be to removing it altogether. But not sure if this serves as a reference.

Reply:
I don't think it's old code.  I think these are `kubebuilder` tags for when the CRDs are generated.  Though looking at the docs, this name change is likely fine: https://book.kubebuilder.io/reference/generating-crd.html#additional-printer-columns

Reply:
Nice information Jeff. I think we can roll it back for now, and leave this change for when we deal with the CRDs.

Reply:
rolled back.